detail: fixes more detailed pages

ioannistsanaktsidis opened the pull request:
* Closes #1627.

 * Closes #1628.

Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

tiborsimko commented:
The order of boxes is not right as the last time... But the most visible thing is "How were these data selected?" which is displayed also for non-data. Example: http://0.0.0.0/record/250.  We should alter the alter heading to something more neutral...  (yeah, if dataset then ... otherwise <neutral term>)

tiborsimko commented:
... the rest of the detailed record page changes in this PR are otherwise OK.

But in the commit log you mention also `closes #3500`, but there is no such issue.  Which one did you meant?

tiborsimko commented on cernopendata/templates/cernopendata_records_ui/terms/detail.html:
```diff
@@ -14,7 +14,7 @@ <h2 class="d-inline">
               <div class="col-md-12">
                 {% for exp in record.experiment %}
-                <span class="badge badge-success">{{exp.name}}</span>
+                <a href="/search?experiment={{exp.name}}"><span class="badge badge-secondary">{{exp.name}}</span></a>
```
Hmm, do we want to remain in the glossary context for these searchable links? Wondering because the other glossary labels work that way... If so, we would have to select also the type glossary for the links. CC @sefeg @daslerr

sefeg replied:
I don't fully understand the question

tiborsimko replied:
Imagine you are viewing http://opendata-dev.web.cern.ch/glossary/primary and you click on the "trigger" button (which leads to another fellow glossary term). What would you expect when you click on "cms"?  See only CMS-related glossary terms, or see everything we have that is CMS related?  

The current PR  would bring you to: 
- http://opendata-dev.web.cern.ch/search?page=1&size=20&experiment=CMS
representing the latter option.

It could also bring you to:
- http://opendata-dev.web.cern.ch/search?page=1&size=20&experiment=CMS&type=Glossary
representing the former option. (after we fix the query...)

Basically, this is about expected label click preferences... Have you seen any preferences in your user tests?

sefeg replied:
Ah alright, I was confused with the term "link", as I rather see them as "tags". And since they are designed as tags, the expectation should be to see all resources related to it. So, I am inclined to agree to the way the current PR deals with the issue.

ioannistsanaktsidis commented:
@tiborsimko +1 for making the text general for all the headings. I think that this can be easily done so as to avoid  all the checks in the templates. The 3500 is indeed wrong. This was the number of the record, the issue that is closing is #1622. I updated the commit message. For the glossary I believe since when you open a detailed record you are still in the search page that the labels should lead in the general one e.g experiment=CMS.

tiborsimko commented:
`#1622` checked as well and looks good 😄

So it's only the order of boxes, the headings, and the possible glossary link preference to be seen... (where people are not seeing being "in the search page" exactly, because they are in `http://0.0.0.0/glossary/primary`...)

ioannistsanaktsidis commented:
@tiborsimko  perfect! I guess the remaining parts can be checked in a separate issue when we have finalised all the detail pages content. LGTM this one and after we finish all the record detail pages we create a PR that addresses your comments :)

tiborsimko commented:
Yes, the "order of boxes" can be finalised later... Just don't forget to keep track of examples that were problematic.
